pre_commit/commands/run.py

```diff
@@ -221,6 +221,7 @@ def _compute_cols(hooks: Sequence[Hook]) -> int:
 
 
 def _all_filenames(args: argparse.Namespace) -> Collection[str]:
+    # FIXME: add post-commit somewhere
```

Comment:
nah this would be wrong -- it should act like post-checkout and the hooks should do their own filename generation.